plugins/modules/packaging/language/pear.py

```diff
@@ -51,6 +57,31 @@
     name: pecl/json_post
     state: present
 
+- name: Install pecl package with expected prompt
+- pear:
+    name: pecl/apcu
+    state: present
+    prompts:
+        - (.*)Enable internal debugging in APCu \[no\]
+
+# Install pecl package with expected prompt and an answer
```

Comment:
Please also adjust this example.

Reply:
The example line 67 have been adjusted as well